src/main/command/birthdaycommands/DeleteBirthdayCommand.ts

```diff
@@ -0,0 +1,49 @@
+import { MessageEmbed } from 'discord.js';
+import { Command } from '../Command';
+import { CommandResult } from '../classes/CommandResult';
+import { CommandArgs } from '../classes/CommandArgs';
+import { deleteBirthday } from '../../modules/birthday/BirthdayDbUtil';
+
+const SUCCESSFUL_COMMANDRESULT = new CommandResult(true);
+const UNSUCCESSFUL_COMMANDRESULT = new CommandResult(false);
+
+export class DeleteBirthdayCommand extends Command {
+    public static readonly NAME = 'DeleteBirthday';
+
+    public static readonly DESCRIPTION = "Deletes the user's birthday.";
+
+    /**
+     * Displays all the users with a birthday on the date given as an argument.
+     *
+     * @param { CommandArgs } commandArgs
+     * @returns CommandResult
+     */
+    public async execute(commandArgs: CommandArgs): Promise<CommandResult> {
+        const { userId, messageReply, server } = commandArgs;
+
+        if (!userId) {
+            await messageReply(this.generateInvalidEmbed());
+            return UNSUCCESSFUL_COMMANDRESULT;
+        }
+
+        deleteBirthday(server.serverId, userId);
```

Comment:
You may consider adding a failing message if nothing was deleted from the db. You can detect this from the return object of the `statement#run` statement.
But I think it's not necessary lol

https://github.com/JoshuaWise/better-sqlite3/blob/master/docs/api.md#runbindparameters---object
^ `statement#run` returns an object with a `changes` key which shows the total number of rows that were inserted, updated, or deleted by this operation.

Reply:
Good point. I'll add it soon.